test/bson_corpus/double.json

```diff
@@ -28,16 +28,16 @@
             "relaxed_extjson": "{\"d\" : -1.0001220703125}"
         },
         {
-            "description": "1.2345678901234568e+18",
-            "canonical_bson": "1000000001640081E97DF41022B14300",
-            "canonical_extjson": "{\"d\" : {\"$numberDouble\": \"1.2345678901234568e+18\"}}",
-            "relaxed_extjson": "{\"d\" : 1.2345678901234568E+18}"
+            "description": "1.2345678921232E+18",
+            "canonical_bson": "100000000164002a1bf5f41022b14300",
+            "canonical_extjson": "{\"d\" : {\"$numberDouble\": \"1.2345678921232e+18\"}}",
+            "relaxed_extjson": "{\"d\" : 1.2345678921232E+18}"
         },
         {
-            "description": "-1.2345678901234568e+18",
-            "canonical_bson": "1000000001640081E97DF41022B1C300",
-            "canonical_extjson": "{\"d\" : {\"$numberDouble\": \"-1.2345678901234568e+18\"}}",
-            "relaxed_extjson": "{\"d\" : -1.2345678901234568e+18}"
+            "description": "-1.2345678921232E+18",
+            "canonical_bson": "100000000164002a1bf5f41022b1c300",
+            "canonical_extjson": "{\"d\" : {\"$numberDouble\": \"-1.2345678921232e+18\"}}",
```

Comment:
The same note as above applies here for the lowercase 'e'.

Reply:
Ack.